backend/api/Database/Models/Inspection.cs

```diff
@@ -50,10 +50,12 @@ public Inspection(Inspection copy, InspectionStatus? inspectionStatus = null)
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public string Id { get; set; }
 
+        [Required]
         [MaxLength(200)]
         // ReSharper disable once AutoPropertyCanBeMadeGetOnly.Local
         public string? IsarStepId { get; private set; } = Guid.NewGuid().ToString();
```

Comment:
Can the question mark in `string?` be removed?

Reply:
Yes actually. It was nullable since we use an empty value when generating hash values, but we now use an empty string anyways so this can be required in C# as well.

Reply:
Then I think we remove it